src/Entity/AdherentMessage/Filter/BasicUserFiltersTrait.php

```diff
@@ -26,21 +26,21 @@ trait BasicUserFiltersTrait
      *
      * @ORM\Column(type="boolean")
      */
-    private $includeCommitteeSupervisors = true;
+    private $includeCommitteeSupervisors;
```

Comment:
the idea behind this change is : if true, add the filter, if false, add the filter but negationned, if null, just skip the filter. It allows us to not rely on a new field. WDYT ?

Reply:
yes, this is a good approach for me